notebooks/01_MNIST_using_padl.ipynb

```diff
@@ -1,5 +1,23 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "49b4084e",
+   "metadata": {},
+   "source": [
+    "# Install `PADL`\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64f0a01e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m pip install git+https://github.com/lf1-io/padl.git"
+   ]
+  },
```

Comment:
Please uncomment the installs for `matplotlib`, `torchvision`, etc. And I think we should remove the `0.11.0` requirement for torchvision because it doesn't match the version of `torch` that is installed which is version `1.10`.